core/local/local_test.go

```diff
@@ -728,3 +728,56 @@ func TestSetPaused(t *testing.T) {
 		require.Contains(t, err.Error(), "doesn't support pause and resume operations after its start")
 	})
 }
+
+func TestNewExecutionSchedulerHasWork(t *testing.T) {
+	t.Parallel()
+	script := []byte(`
+import http from 'k6/http';
```

Comment:
This would look better if it was indented further...

Reply:
Fixed